iped-utils/src/main/java/iped/utils/fsw/FileAttributes.java

```diff
@@ -0,0 +1,69 @@
+package iped.utils.fsw;
+
+import java.nio.file.Files;
+import java.nio.file.NotDirectoryException;
+import java.nio.file.attribute.BasicFileAttributes;
+import java.nio.file.attribute.FileTime;
+
+public class FileAttributes implements BasicFileAttributes {
+
+	PathWrapper path;
+	
+	public FileAttributes(PathWrapper path) {
+		this.path = path;
+	}
+	
+
+	@Override
+	public FileTime lastModifiedTime() {
+		return null;
+	}
+
+	@Override
+	public FileTime lastAccessTime() {
+		return null;
+	}
+
+	@Override
+	public FileTime creationTime() {
+		return null;
+	}
+
+	@Override
+	public boolean isRegularFile() {
+		return path.toFile().isFile();
+	}
+
+	@Override
+	public boolean isDirectory() {
+		try {
+			Files.newDirectoryStream(path);
+        } catch (NotDirectoryException ioe) {
+    		return false;
+        } catch (Exception e) {
+        	return true;
+		}
+		return true;
+	}
```

Comment:
Shouldn't this method have the same logic as FileInputStreamFactory.isDirectory(...) method?